.gitignore

```diff
@@ -1,5 +1,9 @@
 *.class
 
+# Maven target folders
+usng4j-api/target/**
```

Comment:
✏️ could just do `target/` in case we add more modules to this repo in the future